src/System.IdentityModel.Tokens.Jwt/JwtRegisteredClaimNames.cs

```diff
@@ -59,6 +68,11 @@ public struct JwtRegisteredClaimNames
         /// </summary>
         public const string Email = Microsoft.IdentityModel.JsonWebTokens.JwtRegisteredClaimNames.Email;
 
+        /// <summary>
+        /// https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
+        /// </summary>
+        public const string EmailVerified = Microsoft.IdentityModel.JsonWebTokens.JwtRegisteredClaimNames.EmailVerified;
```

Comment:
Please add: phone_number, phone_number_verified.
If you could, put AtHash in order.

Thanks.

Reply:
Nice catch, done. Thanks Brent!